src/reducers/index.js

```diff
@@ -54,8 +54,8 @@ export const appendResource = (state, action) => {
 export const removeResource = (state, action) => {
   const newState = _.cloneDeep(state)
   const reduxPath = action.payload
-  const key = reduxPath.slice(-2)[0]
-  const parentReduxPath = reduxPath.slice(0, reduxPath.length - 2)
+  const key = reduxPath.slice(-1)[0]
```

Comment:
It's odd that this didn't require a change to the test for removeResource.   Do we have a good test there?

Reply:
See the change in __tests__/reducers/index.test.js